unstructured/nlp/patterns.py

```diff
@@ -145,3 +145,8 @@
     r"(?:/[a-z0-9$_@.&+!*\\(\\),%-]*)*"
     r"\.(?:jpg|jpeg|png|gif|bmp)"
 )
+
+# NOTE(klaijan) - only supports one level numbered list for now
+# e.g. 1. 2. 3. or 1) 2) 3), not 1.1 1.2 1.3
+NUMBERED_LIST_PATTERN = r"\d+\.\s(.+)"
```

Comment:
How does this pattern also match list formats other than dots, for ex. 1) with a parenthesis instead of period?

Reply:
Oh I much have failed to save this version. Will edit this. Thanks!